assignment1/date_manipulation.rb

```diff
@@ -0,0 +1,35 @@
+require 'date'
+
+puts 'a. Print current date.'
+
+time = Time.now
+puts time
+
+puts '
+b. Print 12th Jan 2012'
+
+puts Time.new(2012, 1, 12).strftime("%d %b %Y")
+
+puts '
+c. Add 7 days in current date'
+
+puts (time + (3600 * 24 * 7))
+
+puts '
+d. Compare two dates. (12-04-2010 & 12-05-2011). Calculate the days between these two dates.'
+
+date1 = Date.new(2010, 4, 12)
```

Comment:
your inputs should be "12-04-2010" and "12-05-2011" in string format for this method to calculate the difference and you output should be like 7 days, 21 days, 1 day etc